featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -375,6 +379,7 @@ def calc_results(time_last, ids, precalculated_features=None, training_window=No
 
 
 def approximate_features(feature_set, cutoff_time, window, entityset,
+                         approximate_feature_trie,
```

Comment:
get this value from `feature_set`. likely renamed `ignored_feature_trie` in feature_set as well